src/annotate/seqvars/mod.rs

```diff
@@ -535,14 +535,17 @@ pub fn load_tx_db(tx_path: &str) -> Result<TxSeqDatabase, anyhow::Error> {
 }
 
 /// Mehari-local trait for writing out annotated VCF records as VCF or VarFish TSV.
-pub(crate) trait AsyncAnnotatedVariantWriter {
+pub trait AsyncAnnotatedVariantWriter {
+    #[allow(async_fn_in_trait)]
     async fn write_noodles_header(&mut self, header: &VcfHeader) -> Result<(), anyhow::Error>;
+    #[allow(async_fn_in_trait)]
```

Comment:
Potentially use [`asyn_trait`](https://docs.rs/async-trait/latest/async_trait/) here. In any case, the doc should probably be updated, since the trait apparently is not mehari-local ;D